src/dev/ci_setup/setup.sh

```diff
@@ -25,7 +25,7 @@ echo "build --remote_header=x-buildbuddy-api-key=$KIBANA_BUILDBUDDY_CI_API_KEY"
 ### install dependencies
 ###
 echo " -- installing node.js dependencies"
-yarn kbn bootstrap --verbose
+BUILD_TS_REFS_ON_BOOTSTRAP=true yarn kbn bootstrap --verbose
```

Comment:
@brianseeders, there are issues with running `scripts/build_ts_refs` with symlinked node modules, which was attempted [here](https://github.com/elastic/kibana/pull/91234). What would be the best way to go about running that not the main workspace and not through tasks, but not block anything downstream?

Reply:
@tylersmalley You could try creating a new task in `tasks.groovy` that does something like

```groovy
// this could go directly in the tasks array in tasks.check() like this, or you could put it in a separate function
{
  dir("${env.WORKSPACE}/kibana") {
    kibanaPipeline.scriptTask('Build TS Refs', 'test/scripts/jenkins_build_ts_refs.sh')()
  }
}
```

This should let a task process switch back to the original workspace to run the script. Not super great, we have to be careful we don't have multiple things running out of that workspace that could conflict with each other. Other than that, it should get the job done I think.